server/liveblog/blogs/blogs.py

```diff
@@ -51,7 +88,13 @@ def on_create(self, docs):
     def get(self, req, lookup):
         if req is None:
             req = ParsedRequest()
-        return self.backend.get('blogs', req=req, lookup=lookup)
+        docs = super().get(req, lookup)
+        return docs
+
+    def find_one(self, req, **lookup):
+        doc = super().find_one(req, **lookup)
+        get_last_cid(doc)
```

Comment:
Yes, better :)
So I think we can rename this function because it's not a getter, it also updates an attribute.
And what does the acronym `cid` means ?